docs/user-guide/pods.md

```diff
@@ -50,42 +50,92 @@ Documentation for other releases can be found at
 
 <!-- END MUNGE: GENERATED_TOC -->
 
-In Kubernetes, rather than individual application containers, _pods_ are the smallest deployable units that can be created, scheduled, and managed.
+_pods_ are the smallest deployable units that can be created and managed in
+Kubernetes.
 
 ## What is a _pod_?
 
-A _pod_ (as in a pod of whales or pea pod) corresponds to a colocated group of applications running with a shared context. Within that context, the applications may also have individual cgroup isolations applied. A pod models an application-specific "logical host" in a containerized environment. It may contain one or more applications which are relatively tightly coupled &mdash; in a pre-container world, they would have executed on the same physical or virtual host.
+A _pod_ (as in a pod of whales or pea pod) is a group of applications, such as
+Docker containers.  Pods are always co-located and co-scheduled, and run in a
+shared context.  A pod models an application-specific "logical host" - it
+contains one or more applications which are relatively tightly coupled &mdash;
+in a pre-container world, they would have executed on the same physical or
+virtual machine.
+
+The shared context of a pod can be defined as a set of Linux namespaces,
+cgroups, and potentially other facets of isolation.  Within a pod's context,
+the individual applications may have further sub-isolations applied.
+
+Linux namepsaces that are shared across applications in a pod:
+
+* Network namespace: applications within the pod have the same network
+  interfaces, IP addresses, localhost, and port space
+* IPC namespace: applications within the pod can use SystemV IPC or POSIX
+  message queues to communicate with each other
+* UTS namespace: applications within the pod share their hostname
+* PID namespace: applications within the pod can see each other's processes and
+  signal each other
```

Comment:
This seems to much detail about linux namespaces for an introduction to our most basic object.

How about:

Containers in the same Pod share an IP Address and a Hostname.  Containers in different pods always have a different IP address and hostname.
Containers in the same pod can communicate with each other locally (via "localhost", SystemV IPC, Posix Message Queues, and signals).   Containers in different Pods cannot communicate in any of these ways.

See [otherfile] for more details.  Then put the talk about linux namespaces into that other place.